src/algebra/big_operators.lean

```diff
@@ -1537,4 +1538,21 @@ begin
   exact sum_lt_top_iff
 end
 
+open opposite
+
+/-- Moving to the opposite ring commutes with summing. -/
+@[simp] lemma op_sum [ring β] {s : finset α} (f : α → β) :
+  op (∑ x in s, f x) = ∑ x in s, op (f x) :=
```

Comment:
1. You only need `add_comm_monoid β`.
2. Could you please define `op_add_hom : α →+ op α := ⟨op, op_zero, op_add⟩` + `@[simp] lemma coe_op_add_hom : ⇑(op_add_hom : α →+ op α) = op := rfl` in `algebra/opposite`, then use `op_add_hom.map_sum` here?